### PR TITLE
Fix OCPQE-25621

### PR DESCRIPTION
**Pull request by JianLi-RH**

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-upgrade-from-stable-4.13-aws-upi-basecap-none-arm-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-upgrade-from-stable-4.13-aws-upi-basecap-none-arm-f28

**Comment by JianLi-RH** [comment]
hit `argument list too long`, could we merge this PR with out test run? The new caps are all in `supported values: "CSISnapshot", "Console", "Insights", "NodeTuning", "Storage", "baremetal", "marketplace", "openshift-samples"]`

/cc @shellyyang1989 @jiajliu @jianlinliu

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.13.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential CloudControllerManager Ingress
+      ADDITIONAL_ENABLED_CAPABILITIES: Console Insights openshift-samples
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.13.yaml:104]: Per the source version setting (`aws-upi-basecap-none-arm-f28` in openshift-openshift-tests-private-release-4.13__multi-nightly.yaml), the job does NOT need any setting for ADDITIONAL_ENABLED_CAPABILITIES, right?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.13.yaml:104]: I don't remember why this setting was added, just checked https://github.com/openshift/release/pull/53849, there is no comments on it. It seems a mistake.

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.13.yaml]: Removed it

**Pull request merged**